### PR TITLE
Docs--Tag-link-update

### PR DESCRIPTION
**Pull request by RichKummer**
### Related Ticket(s)

#1127

[Website audit - v2 prep](https://ibm.ent.box.com/notes/1016248376301)

### Description

Updated `Tag link` page with new content and removed the notification

### Changelog

**New**

- N/A

**Changed**

- Overview content to add more information.
- Updated the Content guidance from `Copy` to `Label`
- Rename page to `Tags`
- Add `Tag group` section for more info

**Removed**

- Notification on Tag link page.

### CODE REVIEW

File: src/pages/components/tags.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  
+</AnchorLinks>
+
+## Overview
+
+TBD content
+
+## Tag
+
+The Carbon tag is primarily used inside the [Card](https://www.ibm.com/standards/carbon/components/cards) component in Carbon for IBM.com. This enables Cards to display labels that categorize the Card's contents or highlight aspects of the content.
```
Comment (kennylam) [inline src/pages/components/tags.mdx:29]:
```suggestion
The Carbon tag is primarily used inside the [card](https://www.ibm.com/standards/carbon/components/cards) component in Carbon for IBM.com. This enables cards to display labels that categorize the card's contents or highlight aspects of the content.
```

Shouldn't be title-cased, right?

File: src/pages/components/tags.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+For more information on Tag, visit the [Tag guidance](https://carbondesignsystem.com/components/tag/usage/) for the Carbon Design System.
```
Comment (kennylam) [inline src/pages/components/tags.mdx:31]:
```suggestion
For more information, visit Carbon Design System's tag [guidance](https://carbondesignsystem.com/components/tag/usage/) page.
```

File: src/pages/components/tags.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Tag link
+
+Tag link is a clickable element that links the topic to another page for users to find more information about the topic. It uses Carbon's [Tag](https://carbondesignsystem.com/components/tag/usage/) as a base, and uses a larger type token and additional spacing around the label. Unlike Tag, Tag link has limited color styles for the label and background.
```
Comment (kennylam) [inline src/pages/components/tags.mdx:35]:
```suggestion
Tag link is a clickable element that links the topic to another page for users to find more information about the topic. It uses Carbon's [tag](https://carbondesignsystem.com/components/tag/usage/) as a base, and uses a larger type token and additional spacing around the label. Unlike `tag`, `tag link` has limited color styles for the label and background.
```

File: src/pages/components/tags.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Tag link is primarily used inside of a Tag group (see more below).
```
Comment (kennylam) [inline src/pages/components/tags.mdx:37]:
```suggestion
Tag link is primarily used inside of a tag group (see more below).
```

File: src/pages/components/tags.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+### Content guidance for Tag link
+
+| Element | Content type | Required | Instances | Character limit <br/>(English / translated) | Notes               |
+| ------- | ------------ | -------- | --------- | ------------------------------------------- | ------------------- |
+| Label   | Text         | No       | 1         | 25 / 35                                     | URL added in setup. |
+
+## Tag group
+
+Tag group is a collection of Tag links that can be added to the [Lead space](https://www.ibm.com/standards/carbon/components/leadspace). Tag group adds additional context to the Lead space content, and each Tag link can navigate users to a broader category within IBM.com.
```
Comment (kennylam) [inline src/pages/components/tags.mdx:49]:
```suggestion
Tag group is a collection of tag links that can be added to the [lead space](https://www.ibm.com/standards/carbon/components/leadspace). Tag group adds additional context to the lead space content, and each tag link can navigate users to a broader category within IBM.com.
```

**Review by kennylam — COMMENTED** [review]
Thanks Rich! Just some suggestions to remove title-case from component names.

File: src/pages/components/tags.mdx [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+<AnchorLink>Tag</AnchorLink>
+<AnchorLink>Tag link</AnchorLink>
+<AnchorLink>Tag group</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+  
+</AnchorLinks>
+
+## Overview
+
+Tags are elements that add additional context to content that it appears in. In Carbon for IBM.com, the Carbon tag is used directly in the card component. Tag link is a clickable version of the Carbon tag that is often used inside a tag group.
```
Comment (oliviaflory) [inline src/pages/components/tags.mdx:25]:
```suggestion
Tags are used to add additional context to, or categorize content that they appear in. In Carbon for IBM.com, the Carbon tag is used directly in the card component. Tag link is a clickable version of the Carbon tag that is often used inside a tag group to represent poly-hierarchy.
```
Suggestion to remove "element"

**Review by oliviaflory — COMMENTED** [review]
@RichKummer I think this looks good, I added a few minor edits. LMK what you think!

**Review by oliviaflory — APPROVED** [review]
LGTM!

**Pull request merged**